### DISCUSSION TITLE
Aircrack-ng and 5 GhZ OPN ap

### DISCUSSION THREAD

**Discussion by ChillVibesMushroom** [aircrack-ng/aircrack-ng#2329] [Q&A]
Not a defect just a question

I'm capturing traffic on a 5Ghz OPN Network and pulling information from my Android device but all traffic is coming in as TLS 1 1.2 1.3 and typically I'll get DNS traffic and what not but this time I'm having to rely on the client hello from the TLS packets to view traffic information I noticed if I used different commands like

ip.src == 192.168.1.1 then I can get a more focused view in the traffic which shows me some things but it isn't as clean as just typing DNS in Wireshark and boom seeing all app traffic from my Android phone or my iphone.

System information
OS Name: Dragon OS Focal 🐉
OS Version: latest
Wireless card and chipset (if relevant):
Alfa

I noticed that aireplay-ng doesn't kick the client off when targeting the device but will kick the client off when targeting the router itself

WPA2 router

Typically in Wireshark I type DNS MDNS HTTP and get the wanted traffic

How to reproduce the issue
sudo airmon-ng start wlan0

sudo airodump-ng wlan0mon --band a

Selects desired network

Runs Wireshark

Connects my phone

Browses apps and information is gathered 

Also, Packet injection works

On the protected network 5G in the same place I can gather info using the AP password and capturing a half handshake 

Related issues

I've been looking into ways to decrypt TLS

I found one method where we create a keylog file and from the same shell run Firefox but I need something OTA and I was told it's possible just not how

I came here because I know you can help

I was thinking of doing possible karma attacks if you guys couldn't give me a clear answer but even then what if the AP doesn't allow dissociation.

What I want to know is how do I decrypt TLS info most of the packets seem to say 802.11 lol what am I doing wrong? I'm using the same methods as before should I reset Wireshark and how would I do that?

**Comment by Mister-X-** [comment]
Please fix your formatting and make it so it is readable. Not just copy and pasting from your "issue".

**Reply by ChillVibesMushroom** [reply]
> Please fix your formatting and make it so it is readable. Not just copy and pasting from your "issue".

Sorry this all happened today after the gym I was really tired but decided to hit up one of my favorite spots

**Reply by ChillVibesMushroom** [reply]
Would you like to see the pcap file ? I have it saved it's very strange tbh imho

**Comment by Mister-X-** [comment]
Please use markdown formatting (such as command, titles, bullet points, etc), provide version numbers instead of 'latest', and a link to the OS to download.

If you aren't making any effort, why should I?

**Reply by ChillVibesMushroom** [reply]
I'll do it when I have the time right now I'm going to work so I don't have time have a goodnight though

**Reply by ChillVibesMushroom** [reply]
> Please use markdown formatting (such as command, titles, bullet points, etc), provide version numbers instead of 'latest', and a link to the OS to download.
> 
> If you aren't making any effort, why should I?

I did write out the commands I used

**Reply by ChillVibesMushroom** [reply]
OS is dragon os focal 20.04.2
Aircrack-ng version is 1.6

https://sourceforge.net/projects/dragonos-focal/

I'll upload the pcap file to look at later on in the day when I have time as I'm about to go to work but I did study up a little bit on TLS and I noticed we can still pull some information from the Wireshark pcap file while using airodump-ng do you think it might be a router thing? I noticed the ip addresses are random they don't follow the natural order of 192.168.1.1 192.168.1.2 192.168.1.3


They'll go 192.168.1.12 192.168.5.2 etc...

**Comment by Mister-X-** [comment]
1. Update to 1.7, 1.6 is old.
2. Which alfa adpater? give the output of airmon-ng.
